File: src/main/java/com/leijendary/spring/boot/template/core/data/TranslationRequest.java
package com.leijendary.spring.boot.template.core.data;

import javax.validation.constraints.Min;
import javax.validation.constraints.NotBlank;

import lombok.Data;

@Data
public class TranslationRequest {

    @NotBlank(message = "validation.required")
    private String language;

    @Min(message = "validation.min", value = 1)
    private int ordinal;
}

File: src/main/kotlin/com/leijendary/spring/boot/template/core/data/TranslationRequest.kt
package com.leijendary.spring.boot.template.core.data

import javax.validation.constraints.Min
import javax.validation.constraints.NotBlank

open class TranslationRequest(
    @get:NotBlank(message = "validation.required")
    val language: String? = null,

    @get:Min(value = 1, message = "validation.min")
    val ordinal: Int = 0
)